src/pages/Products/Products.scss

```
@@ -0,0 +1,247 @@
+            flex-direction: column;
+            z-index: 100;
+          }
+        }
+      }
+
+      .rightColumn {
+        width: 46%;
+        max-width: 328px;
+        flex-wrap: wrap;
+
+        .textBadge {
+          font-size: 12px;
+          font-weight: 600;
+          font-family: 'Noto Sans KR', sans-serif;
+        }
+
+        .productNameEn {
+          @include fontMyeongjo;
+          font-size: 24px;
+          line-height: 1.3;
+          margin-top: 20px;
+        }
+
+        .productNameKr {
+          @include fontMyeongjo;
+          font-size: 20px;
+          line-height: 24px;
+        }
+
+        .price {
+          color: $color-black;
+          margin: 10px 0 24px 0;
+          font-size: 12px;
+          font-weight: 600;
+          line-height: 1;
+        }
+
+        .firstReview {
+          @include fontUnderline;
+          display: flex;
+          justify-content: flex-end;
+          margin-left: 10px;
+          margin-bottom: 18px;
+          font-size: 12px;
+          font-weight: 600;
+        }
+
+        .description {
+          display: inline;
+          font-size: 12px;
+          font-weight: 400;
+          letter-spacing: 0.04em;
+          line-height: 1.6;
+          margin-top: 16px;
+          margin-bottom: 26px;
+        }
+
+        .btnMore {
+          @include fontUnderline;
+          font-size: 12px;
+          font-weight: 700;
+          border: none;
+          background: 0 0;
+        }
+
+        .sizeThumbnails {
+          display: flex;
+
+          .sizePicker {
+            display: inline-block;
+
+            img {
+              width: 58px;
+              height: inherit;
+            }
+          }
+        }
+
+        .btn {
+          background-color: rgb(33, 33, 33);
+          color: #fff;
+          width: 100%;
+          height: 50px;
+          font-size: 12px;
+          letter-spacing: 1.25px;
+          padding: 0px 16px;
+          margin: 24px 0 30px;
+          overflow-x: hidden;
+          cursor: pointer;
+        }
+
+        .btnWish {
+          display: flex;
+          justify-content: flex-end;
+          padding: 1px 6px;
+          font-size: 12px;
+          font-weight: 400;
+          border: none;
+          background: 0 0;
+        }
+      }
+      .contents {
+        width: 100%;
+        margin-top: 50px;
+
+        h3 {
+          font-size: 12px;
+          font-weight: 400;
+          line-height: 1.7;
+          margin: 0 48px;
+          padding-top: 20px;
+        }
+
+        .ingredients {
+          border-bottom: 1px solid #e2e1d7;
+          border-top: 1px solid #e2e1d7;
+          justify-content: space-between;
+          align-items: center;
+
+          h2 {
+            @include fontMyeongjo;
+            font-size: 20px;
+            font-weight: 400;
+            padding: 46px 0;
+            display: flex;
+          }
+
+          p {
+            padding-top: 12px;
+            padding-bottom: 40px;
```

```suggestion
            padding: 12px 0 40px;
```
위에 margin에 주셨던 방법처럼 이 부분도 변경이 가능하겠습니다!